src/main/scala/no/ndla/searchapi/model/search/SearchableLanguageFormats.scala

```diff
@@ -0,0 +1,122 @@
+/*
+ * Part of NDLA search_api.
+ * Copyright (C) 2018 NDLA
+ *
+ * See LICENSE
+ */
+
+package no.ndla.searchapi.model.search
+
+import java.util.{Date, TimeZone}
+
+import no.ndla.searchapi.model.domain.article.LearningResourceType
+import no.ndla.searchapi.model.domain.{LanguagelessSearchableTaxonomyContext, SearchableTaxonomyContext}
+import no.ndla.searchapi.model.taxonomy.{ContextFilter, SearchableContextFilters}
+import org.joda.time.{DateTime, DateTimeZone}
+import org.json4s.JsonAST.{JField, JObject}
+import org.json4s.{CustomSerializer, Extraction}
+import org.json4s._
+
+class SearchableArticleSerializer
+    extends CustomSerializer[SearchableArticle](_ =>
+      ({
+        case obj: JObject =>
+          implicit val formats: Formats = org.json4s.DefaultFormats + new TaxonomyContextSerializer ++ org.json4s.ext.JodaTimeSerializers.all
+
+          val time = (obj \ "lastUpdated").extract[DateTime]
+          val tz = TimeZone.getDefault
+          val lastUpdated = new DateTime(time, DateTimeZone.forID(tz.getID))
+
+          val x = SearchableArticle(
+            id = (obj \ "id").extract[Long],
+            title = SearchableLanguageValues("title", obj),
+            content = SearchableLanguageValues("content", obj),
+            visualElement = SearchableLanguageValues("visualElement", obj),
+            introduction = SearchableLanguageValues("introduction", obj),
+            metaDescription = SearchableLanguageValues("metaDescription", obj),
+            tags = SearchableLanguageList("tags", obj),
+            lastUpdated = lastUpdated,
+            license = (obj \ "license").extract[String],
+            authors = (obj \ "authors").extract[List[String]],
+            articleType = (obj \ "articleType").extract[String],
+            defaultTitle = (obj \ "defaultTitle").extract[Option[String]],
+            metaImageId = (obj \ "metaImageId").extract[Option[Long]],
+            supportedLanguages =
+              (obj \ "supportedLanguages").extract[List[String]],
+            contexts = (obj \ "contexts").extract[List[SearchableTaxonomyContext]]
+          )
+          x
```

Comment:
`x`?

Reply:
Hatt denne i variabel for enklere debugging. Fikser :)